pipeline/frontend/yaml/stepbuilder/stepBuilder.go

```diff
@@ -33,32 +32,29 @@ import (
 	"go.woodpecker-ci.org/woodpecker/v2/pipeline/frontend/yaml/linter"
 	"go.woodpecker-ci.org/woodpecker/v2/pipeline/frontend/yaml/matrix"
 	yaml_types "go.woodpecker-ci.org/woodpecker/v2/pipeline/frontend/yaml/types"
-	"go.woodpecker-ci.org/woodpecker/v2/server"
 	forge_types "go.woodpecker-ci.org/woodpecker/v2/server/forge/types"
```

Comment:
pipeline now depends on server :/

Reply:
As you might have seen in the code below: `// TODO: use don't import from server => move FileMeta to pipeline package` 😉

Reply:
Duplicating server models is once again quite a pain. Maybe we should move the models to a central place as duplicating them as we did in `woodpecker-go` for example is making the code mainly harder to update and read

Reply:
No as yes there is shared code but ... server uses both frontend types and backend type like definitions ... once #2851 and similar are finished ... we could just add linting to have servermodels be equal to frontend types.

Still the engine should not have any domain knowledge about e.g. sql ... but the server models do

Yes it looks bit clustered sometimes ... but I wana have woodpecker to asemble at least some software design pattern to have it easyer to extend in the long run ...